console/EAP7-837_Customize_Management_Console_Title.adoc

```diff
@@ -0,0 +1,54 @@
+= Expose EJB runtime data / statistics
+:author:            Harald Pehl
+:email:             hpehl@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/
+
+== Overview
+
+Customise the title of the management console. A custom title can be specified in the user settings of the console. The title can contain the following placeholders:
+
+- `%n`: The `name` attribute of the the root resource
+- `%o`: The `organization` (standalone) resp. `domain-organization` attribute of the the root resource
+
+If the title is not specified, the built in title will be used. If a placeholder is used, but the attribute is undefined, the built in title will be used.
```

Comment:
The title will be stored as cookie (just like the other settings). For the long-term this might change and settings and other data could end up in a console subsystem.

Reply:
I am just thinking if migration of such settings into "console subsystem" won't it be a problem for users? Settings won't be "per-user" anymore but rather "per-instance". What are the obstacles in introducing that subsystem now?